doc/antora/modules/reference/nav.adoc

```diff
@@ -128,108 +128,108 @@
 ** xref:raddb/mods-available/index.adoc[Modules]
 *** xref:raddb/mods-available/all_modules.adoc[Summary of all modules]
 *** xref:raddb/mods-available/abfab_psk_sql.adoc[ADFAB PSK]
-*** xref:mods-available/always.adoc[Always]
-*** xref:mods-available/attr_filter.adoc[Attr_filter]
-*** xref:mods-available/cache.adoc[Cache]
-*** xref:mods-available/cache_eap.adoc[Cache EAP]
-*** xref:mods-available/cache_tls.adoc[Cache TLS Session]
-*** xref:mods-available/chap.adoc[CHAP module]
-*** xref:mods-available/cipher.adoc[Cipher]
-*** xref:mods-available/client.adoc[Client]
-*** xref:mods-available/couchbase.adoc[Couchbase]
-*** xref:mods-available/csv.adoc[CSV]
-*** xref:mods-available/cui.adoc[CUI]
-*** xref:mods-available/date.adoc[Date]
-*** xref:mods-available/delay.adoc[Delay]
-*** xref:mods-available/detail.adoc[Detail]
-*** xref:mods-available/detail.example.com.adoc[Detail (Sample)]
-*** xref:mods-available/detail.log.adoc[Detail (Log Sample)]
-*** xref:mods-available/dhcpv4.adoc[DHCPv4]
-*** xref:mods-available/dict.adoc[Dict]
-*** xref:mods-available/digest.adoc[Digest]
-*** xref:mods-available/eap.adoc[EAP]
-*** xref:mods-available/eap_inner.adoc[EAP/Inner]
-*** xref:mods-available/echo.adoc[Echo]
-*** xref:mods-available/escape.adoc[Escape]
-*** xref:mods-available/etc_group.adoc[etc_group]
-*** xref:mods-available/exec.adoc[Exec]
+*** xref:raddb/mods-available/always.adoc[Always]
```

Comment:
I would prefer one commit for all of the changes, rather than multiple "back and forth" commits.